stackit/internal/services/ske/cluster/resource.go

```diff
@@ -587,6 +601,21 @@ func (r *clusterResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 							},
 						},
 					},
+					"dns": schema.SingleNestedAttribute{
+						Description: "DNS extension configuration",
+						Computed:    true,
+						Attributes: map[string]schema.Attribute{
+							"enabled": schema.BoolAttribute{
+								Description: "Flag to enable/disable DNS extensions",
+								Computed:    true,
+							},
+							"zones": schema.ListAttribute{
+								Description: "Specify a list of domain filters for externalDNS (e.g., `foo.runs.onstackit.cloud`)",
+								Computed:    true,
+								ElementType: types.StringType,
+							},
+						},
+					},
```

Comment:
Are these fields really just `Computed`? Shouldn't they be `Optional` to allow the user to set them?

Reply:
Fixed